common.sh

```diff
@@ -145,6 +149,14 @@ function build_kb {
   GO111MODULE=on go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
 }
 
+function fetch_kind {
+  header_text "Checking for kind"
+  if ! is_installed kind ; then
+    header_text "Installing kind"
+    GO111MODULE=on go get sigs.k8s.io/kind@v0.5.1
```

Comment:
I was recently bit by this behavior. `go get` will modify `go.mod` file if `go get` fetches new deps and also introduces indirect dependency on the project. So the work around is to run `go get` from a temporary directory instead of kb directory.

Reply:
We have install_and_setup.sh under `scripts` that is used in travis setup for setting up kind cluster. Would be good if we can have common code for installing kind so that we don't end up with two different ways of installing kind.

Reply:
how's that?